lib/core/services/people-content.service.ts

```diff
@@ -69,6 +75,14 @@ export class PeopleContentService {
         );
     }
 
+    async isContentAdmin(): Promise<boolean> {
+        if (this.hasContentAdminRole === undefined) {
+            const user: PersonEntry = await this.getCurrentPerson().toPromise();
+            this.hasContentAdminRole = user?.entry?.capabilities?.isAdmin;
```

Comment:
assign only if is present 
user?.entry?.capabilities?.isAdmin ? true : false

Reply:
isn't it the same? 
The IDE is suggesting to keep it like this `user?.entry?.capabilities?.isAdmin;` as isAdmin is a boolean
<img width="911" alt="Screenshot 2021-04-23 at 11 54 05 am" src="https://user-images.githubusercontent.com/32884230/115861432-b22ff080-a42a-11eb-8ca2-f5837c6089e5.png">

Reply:
what if the entry does not exist ? ok most probably it will return false not undefined